clone_update_repo.sh

```diff
@@ -1,79 +1,118 @@
+#!/bin/bash
+
+# Added color variables
+txtrst=$(tput sgr0) 
+txtred=$(tput setaf 1)
+
 # Add announcement that this script has just started with timestamp
-# TODO
+echo "${txtred}Timestamp:${txtrst} Script has started at $(date +%T)"
 
 # Clone the "happy_bank_core" repo from Github
-# TODO
+echo "${txtred}Cloned remote repository:${txtrst}"
+git clone https://github.com/JanMate/happy_bank_core.git
+cd happy_bank_core
 
 # Create a new branch "update-readme" and checkout it
-# TODO
+echo "${txtred}Created new branch:${txtrst}"
+git checkout -b "update-readme"
+git checkout update-readme #already on this branch
 
 # Update README in cloned repo - copy there the differences of the file in this repo
 # Commit message and Workflow sub-sections (Hint: Try to use "head" or "tail" command + pipe "|" to redirect output)
-# TODO
+echo "${txtred}Updated content of the README.md:${txtrst}"
+infra_readme_url=https://raw.githubusercontent.com/JanMate/happy_bank_infra/main/README.md
+curl -s "$infra_readme_url" | sed -n "4,79p" > README.md
```

Comment:
Hi Jan,  Thank you for the explanation, I was struggling a lot with this one. I used this approach with the link to the raw file.  Is it okay?

Reply:
You don't need to download any files are part of this repo. Just use them.